### DISCUSSION TITLE
Cooja tooltips

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#1998] [Q&A] [answered]
Cooja will display yellow tooltips in the Mote output window when the log lines are longer than the window width. Sometimes it is very useful, sometimes not. Is there a way to configure whether I want the tooltips to be displayed?

**Comment by rahav** [comment]
Another naïve question: Can we remove the right hand pane in Cooja (the one with the help text) from being displayed? It takes important display real estate and it would be great if we can view it only when needed.

**Reply by nfi** [reply]
The right hand pane with help text can be toggled on/off by pressing `F1` or selecting `Quick help` in the top menu `Help`.

**Comment by nfi** [comment, marked as answer]
There is currently no option to hide the tooltip in the mote output, but I think adding such an option would be a minor change.

**Reply by rahav** [reply]
Thank you.
